tests/test_sign_with_hsm/setup.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# Copyright 2022 Northern.tech AS
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+set -ex
+
+export TEST_ROOT=$(dirname "$0")
+[[ -f ${TEST_ROOT}/functions.sh ]] && . ${TEST_ROOT}/functions.sh
+
+is_debian && debian_setup
```

Comment:
I suggest we completely remove the `is_debian` test. I don't want to take any risk in the test not being run and going unnoticed. It should only be called by CI on a debian based image... so as far as the test is concern it should just run and fail if something doesn't work.

Reply:
I added an explicit `exit 1` in case we are running on non Debian. I want to keep `is_debian` logic, if that does not bother you. the test will now fail as you suggest.